common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -36,43 +36,37 @@
 @FeatureInfo(category = FeatureCategory.OVERLAYS)
 public class CustomBarsOverlayFeature extends UserFeature {
 
-    @Config
-    public boolean shouldDisplayOnActionBar = false;
-
-    @Config
-    public boolean shouldDisplayOnBossBar = false;
-
     @Override
     public List<Class<? extends Model>> getModelDependencies() {
         return List.of(ActionBarModel.class, BossBarModel.class);
     }
 
     @SubscribeEvent
     public void onActionBarManaUpdate(ActionBarMessageUpdateEvent.ManaText event) {
-        if (shouldDisplayOnActionBar || !manaBarOverlay.isEnabled()) return;
+        if (!manaBarOverlay.isEnabled() || ((BaseBarOverlay) manaBarOverlay).shouldDisplayOriginal) return;
```

Comment:
Instead of this cast, use proper Types at the overlay instance creations (line 80-93).

Reply:
I can not test if proper types work ass it wont build correctly

Reply:
Yeah.., I think it should. We have to wait for jcentral anyways, so..

Reply:
(btw `./gradlew buildDependents --offline` works if you really want to test smth)

Reply:
Reminder that you can test it now :)